k8s/openebs-operator.yaml

```diff
@@ -96,12 +96,20 @@ metadata:
   name: openebs-provisioner
   namespace: default
 spec:
-  replicas: 1
+  replicas: 2
   template:
     metadata:
       labels:
         name: openebs-provisioner
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              matchExpressions:
+                key: node
+                operator: Equal
+                values: ["minion01","minion02"]
```

Comment:
This forces the admin to have above values i.e. minion01 & minion02 in his setup.

Reply:
Following is a better feature than labelling the nodes directly.
https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity-beta-feature

Reply:
@satyamz Yes. I was pointing at podAntiAffinity with labels on Pod itself.

Reply:
Yes! I am making/testing changes. Once done, will update PR

Thanks